src/test/config.sol

```diff
@@ -330,7 +332,7 @@ contract Config {
             line:         0,
             dust:         25 * THOUSAND,
             pct:          9_75,
-            mat:          13000,
+            mat:          15000,
```

Comment:
Same as above: I'm not sure why this needed to be changed.

Reply:
Same as above:
https://github.com/makerdao/spells-mainnet/blob/219715be3e4653ff53cd4d6a47b8313fa6720f4e/archive/2024-10-04-DssSpell/DssSpell.sol#L282-L292